ndbench-core/src/main/java/com/netflix/ndbench/core/discovery/AWSLocalClusterDiscovery.java

```diff
@@ -0,0 +1,47 @@
+package com.netflix.ndbench.core.discovery;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AWSLocalClusterDiscovery implements IClusterDiscovery {
+    private static final Logger logger = LoggerFactory.getLogger(LocalClusterDiscovery.class.getName());
+
+    @Override
+    public List<String> getApps() {
+        return Arrays.asList(getLocalhostName());
+    }
+
+    private String getLocalhostName() {
+    	String url = "http://169.254.169.254/latest/meta-data/public-hostname";
```

Comment:
@vinaykumarchella @ipapapa 

Just for sake of curiosity, what would we need to change in order to have this provider working on the VPC env? Or why do you think this code does not work? 

I'm running on AWS on VPC (I dont have access to classic) noddy have anymore only old aws customers. This code is working for me now.... My NDbench instance is running on AWS Cloud with no issues.  I'm just curious.  :-)